### DISCUSSION TITLE
Multiple input keys to one class variable

### DISCUSSION THREAD

**Discussion by vanvanni** [spatie/laravel-data#489] [General]
Hi!

I would like to know how I can assign multiple input keys to this variable. Mostly there is only one present. if not maybe the order of keys decide the importance. Is this possible?

```
class Person extends Data
{
    public function __construct(
        // Input key can be: NAME, name, Name and FIRST_NAME
        public string $name,
    )
    {
    }
}
```

**Comment by rubenvanassche** [comment]
Sadly no, this opens a can of worms in validation so at the moment such a thing is not possible.
